### ISSUE TITLE
Parts of the documentation not indexed for searching (on-site, google, ...)

### ISSUE THREAD

**Issue by BerlinJoker** [home-assistant/home-assistant.github.io#35934]
### Feedback

It seems that parts of the documentation on https://www.home-assistant.io/ are excluded from being indexed for search purpuses. Concretely: the data attributes for actions. My guess is that using the code formatting has a pre-defined "ignore for search" flag set or something. Which makes sense when trying to exclude example code/snippets, but is unfortunate when actual documentation is ignored as well. 

Example: https://www.home-assistant.io/integrations/light/#action-lightturn_on

There e.g. the data attribute **"brightness_step_pct"** is listed. 

![brightness_step_pct](https://github.com/user-attachments/assets/4b104874-50e4-47e5-b057-3ae19db30a75)

When I search on google for 

**home assistant light "brightness_step_pct" site:www.home-assistant.io**

I only get 2 blog entries from when that parameter was introduced, but not the actual light documentation page that I should get.  

![brightness_step_pct_google](https://github.com/user-attachments/assets/72860844-9a07-45e1-91d0-8141b1c6e8c4)

When I search for "brightness_step_pct" in the in-documentation-search, I get zero results. 

![brightness_step_pct_search](https://github.com/user-attachments/assets/8a099f2e-535e-41f7-9075-59a27fc08dda)

Another example that allows narrowing things down a bit: when searching for "color_temp_kelvin" via the in-documentation-search, I get zero results. When I search for "color_temp" only, I get a list of entries, the top result being https://www.home-assistant.io/integrations/yeelight/
There the string "color_temp" is part of a header (https://www.home-assistant.io/integrations/yeelight/#action-yeelightset_color_temp_scene), and not in the code-like formatting as in the table in the general light documentation page.

Long story short: it seems as if parts of the documentation are only of use when you already know which page you need in the first place

### URL

https://www.home-assistant.io/integrations/light/#action-lightturn_on

**Comment by BerlinJoker** [comment]
Sorry to the bot-marked people, I only used your integration(s) as the examples where I ran into the issue. No idea if I myself can nudge the ticket into the right direction and remove you from the assignment....

**Comment by BerlinJoker** [comment]
Just doublechecked,  the issue is still unresolved. And I still think that the documentation should be fully indexed and searchable.
